### DISCUSSION TITLE
Option to disable/remove campaign level tracking

### DISCUSSION THREAD

**Discussion by romainmenke** [plausible/analytics#2306] [Anything Else]
We would prefer it if campaign related url params added by 3rd parties were not send to Plausible.
Is this something that can be configured in the client side script?

**Comment by RobertJoonas** [comment]
Hi @romainmenke!

> Is this something that can be configured in the client side script?

Yes, it is. You should be able to use our `script.manual.js` [script extension](https://plausible.io/docs/script-extensions) to override the URL property sent from the client-side.

Our backend will read all campaign-level data that you send us by default, from the "u" (URL) field. Here's a quick example JSON payload of a Plausible event request (i.e. data of a single event)

```
{
  "n": "pageview",
  "u": "https://plausible.io/blog?utm_campaign=email&something_else=123",
  "d": "plausible.io",
  ...
}
```

The `script.manual.js` extension allows you to override the URL. That should be done with JavaScript. The simplest solution for this use case would be stripping all query parameters from the URL. The solution is very similar to tracking custom query parameters, which we've documented here: https://plausible.io/docs/custom-query-params.

You can actually even follow the same instructions. The only little difference is that you should give an empty list as an argument to the `prepareUrl` function like this:

```javascript
plausible('pageview', { u: prepareUrl([]) })
```

I hope this helps!

**Reply by romainmenke** [reply]
This is exactly what we were looking for, thank you @RobertJoonas for the speedy reply.
